plugins/indexing/sqs/sqs_client.go

```diff
@@ -82,21 +82,22 @@ func (sc *SqsClient) PublishToQueue(height int64, data []*types.Message) error {
 	return nil
 }
 
-func NewSqsClient() *SqsClient {
+func NewSqsClient() (*SqsClient, error) {
 	queueURL, found := os.LookupEnv(queueURLEnvName)
 	if !found {
-		panic(fmt.Errorf("missing environment variable '%s'", queueURLEnvName))
+		err := fmt.Errorf("missing environment variable '%s'", queueURLEnvName)
+		return nil, err
```

Comment:
can be just `nil, fmt.Errorf("missing environment variable '%s'", queueURLEnvName)`